CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # nf-core/bamtofastq: Changelog
 
+## v1.2.0 - Anna Winlock
+
+- [#36](https://github.com/qbic-pipelines/bamtofastq/pull/36) Add options `--cram_files` and `--reference_fasta` to add support for CRAM files.
+
+## v1.1.1 - Katherine Johnson faster
```

Comment:
```suggestion
```
I realise we never actually released this apparently, so this release will contain the speed up too